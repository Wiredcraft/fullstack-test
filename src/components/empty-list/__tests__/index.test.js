import React from 'react';
import renderer from 'react-test-renderer';

import EmptyList from '../index';

jest.mock('react-router-dom');

test('EmptyList renders', () => {
  const component = renderer.create(<EmptyList />);

  expect(component.toJSON()).toMatchSnapshot();
});
