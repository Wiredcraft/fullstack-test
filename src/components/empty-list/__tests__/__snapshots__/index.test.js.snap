// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyList renders 1`] = `
<div
  className="sc-bdvvtL cdNifx"
>
  <p
    className="sc-gsDKAQ liIgxQ"
  >
    No talks yet... Be the first!
  </p>
  <button
    className="sc-dkPtRN bIiLjj"
    onClick={[Function]}
  >
    Add your lightning talk
  </button>
</div>
`;
